frontend/src/common/styles/button.css

```diff
@@ -0,0 +1,12 @@
+.button {
+    background-color: #3388c7;
+    border-radius: 0.5rem;
+    color: white;
+    font-weight: bold;
+    width: 10rem;
+    margin: auto;
+}
```

Comment:
You have 3 options:
1) remove background-color and color, then use color and variant for particular case https://mui.com/material-ui/react-button/#color
2) remove background-color and colors, then create new common styles for confirm and cancel buttons which extend your custom `button` style and set background-color and color
3) don't use custom styles at all, variant="contained" color="success" or "error" is pretty much solve your problem

If you don't like default colors "success", "error", "primary", you can rewrite it or add new https://mui.com/material-ui/customization/palette/#custom-colors